src/EmbeddingSpMDM.cc

```diff
@@ -1377,6 +1377,7 @@ void compressed_indices_remap(
 
   const inst_set_t isa = fbgemmInstructionSet();
   if (isZmm(isa)) {
+#ifndef __HIP_PLATFORM_HCC__
```

Comment:
I believe this change is also related to AVX-512. I might overlook the conclusion of the AVX-512 discussion, but I feel this part is irrelevant to "HIP" or "AMD GPUs".

Reply:
We cannot enable the piece of code block unless the other avx512-related problems are resolved. Otherwise, an "undefined symbol" error is thrown when python imports 

> fbgemm_gpu.